File: Final Project/Food Order Application/Food Order Application Final Project/Food Order Application Final Project/Model/Order.cs
using System.ComponentModel.DataAnnotations;

namespace Food_Order_Application_Final_Project.Model
{
    public class Order
    {
        [Key]
        public int OrderId { get; set; }
       
        public string? CustomerName { get; set; }

        public string? CustomerEmail { get; set; }

        public string? CustomerPhone { get; set; }

        public string? CustomerPlatNo { get; set; }

        public string? CustomerStreet { get; set; }  
        public string? CustomerCity { get; set; }
        public string? CustomerState     { get; set; }

        public float TotalPrice { get; set; }

        public ICollection<Delivery>? Deliveries { get; set; }

    }
}

File: Final Project/Food Order Application/Angular/src/app/order.ts
import { Delivery } from "./Delivery";

export interface Order {
    orderId: number;
    customerName: string;        // Optional property
    customerEmail?: string;       // Optional property
    customerPhone: string;       // Optional property
    customerPlatNo: string;      // Optional property
    customerStreet: string;      // Optional property
    customerCity: string;        // Optional property
    customerState: string;       // Optional property
    totalPrice: number;
    deliveries: any[]; 
        // Optional property, Delivery interface should also be defined
  }